fenix/app/src/main/res/values/strings.xml

```diff
@@ -2235,9 +2235,9 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check_1">Open review checker</string>
     <!-- Content description (not visible, for screen readers etc.) for closing browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check_close">Close review quality check</string>
+    <string name="browser_menu_review_quality_check_close_1">Close review checker</string>
```

Comment:
The `browser_menu_` prefix is a bit unfortunate here since these strings don't actually appear in the browser menu.

Reply:
Should we update the string name to "review_quality_check_close" instead?

Reply:
Please don't, these strings are being translated at this point, and this would require retranslation (for the 3rd time).